Framework/Geometry/test/ObjectTest.h

```diff
@@ -859,6 +859,17 @@ class ObjectTest : public CxxTest::TestSuite {
                     expected, satol);
   }
 
+  void testVolumeUnitCube() {
+    Object_sptr cube = createUnitCube();
+    TS_ASSERT_DELTA(cube->volume(), 1.0, 1e-6)
+  }
+
+  void testVolumeSmallCappedCylinder() {
+    Object_sptr cylinder = createSmallCappedCylinder();
+    const double correctVolume = M_PI * 0.005 * 0.005 * 0.003;
+    TS_ASSERT_DELTA(cylinder->volume(), correctVolume, 1e-6)
+  }
+
```

Comment:
The test unit cube is generated using six planes in `ObjectTest::createUnitCube()`, if I read the code correctly, and this prevents `Object::volume()` recognizing it as a cuboid.